whois-commons/src/main/resources/nrtm_schema.sql

```diff
@@ -19,58 +19,78 @@ create table `source`
 ) ENGINE = InnoDB
   DEFAULT CHARSET = utf8;
 
-DROP TABLE IF EXISTS `version_information`;
-create table `version_information`
+DROP TABLE IF EXISTS `version`;
+create table `version`
 (
     `id`             int unsigned NOT NULL AUTO_INCREMENT,
     `source_id`      int unsigned NOT NULL,
-    `version`        int unsigned NOT NULL DEFAULT '0',
-    `session_id`     varchar(128) NOT NULL DEFAULT '',
-    `type`           varchar(128) NOT NULL DEFAULT '',
-    `last_serial_id` int          NOT NULL DEFAULT '0',
+    `version`        int unsigned NOT NULL,
+    `session_id`     varchar(128) NOT NULL,
```

Comment:
we are mapping a FOREIGN KEY version_id  from notification_file table to id here, what is use of version column?

Reply:
that contains the version number -- "version" in the JSON, so i used the same name here.

Reply:
Too many version names to keep track of now ;)  There is version table name with id, version column and version_id mapping to version. But if it all makes sense I do not mind :)